extensions/huawei/GLSL_HUAWEI_subpass_shading.txt

```diff
@@ -0,0 +1,86 @@
+Name
+
+    HUAWEI_subpass_shading
+
+Name Strings
+
+    GL_HUAWEI_subpass_shading
+
+Contact
+
+    Hueilong Wang (wyvernathuawei), HUAWEI
+
+Contributors
+
+    Hueilong Wang, HUAWEI
+
+Status
+
+    Draft
+
+Version
+
+    Last Modified Date: 2021-04-26
+    Revision: 1
+
+Dependencies
+
+    This extension can be applied to OpenGL GLSL versions 4.60
+    (#version 460) and higher.
+
+    This extension is written against revision 7 of the OpenGL Shading Language
+    version 4.60, dated July 10, 2019.
+
+    This extension interacts with revision 43 of the GL_KHR_vulkan_glsl
+    extension, dated October 25, 2017.
+
+Overview
+
+    This extension allows a subpass shading shader to delcare subpass inputs and use subpass input functions.
```

Comment:
I find "subpass shading shader" quite awkward. Do you prefer that term to the simpler "subpass shader"?

Reply:
I've tried but finally I still have to accept it since it pulls the strings to other naming like pipelines, stages.

Reply:
OK, fair enough.